package.json

```diff
@@ -13,7 +13,7 @@
 		"ui:build": "cd packages/ui/ && pnpm run build",
 		"ui:serve": "cd packages/ui/ && pnpm run serve",
 		"ui": "pnpm run ui:dev",
-		"build": "pnpm run babel -r && pkg .",
```

Comment:
This PR contains previous commits for some reason. 

Maybe try finding ways to get the latest changes only.